Update create-sql-api-dotnet.md

jierong opened the pull request:
The code in line 100 in this doc is wrong, need to remove duplicated new DocumentCollection

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @ganesr, has been notified to review your proposed change.

ganesr commented:
#sign-off

jierong committed:
Update create-sql-api-dotnet.md

The code in line 100 in this doc is wrong, need to remove duplicated new DocumentCollection
